closure/repositories.bzl

```diff
@@ -744,23 +744,24 @@ def com_google_protobuf():
 def com_google_protobuf_js():
     http_archive(
         name = "com_google_protobuf_js",
-        strip_prefix = "protobuf-3.14.0/js",
-        sha256 = "d0f5f605d0d656007ce6c8b5a82df3037e1d8fe8b121ed42e536f569dec16113",
+        build_file = "@io_bazel_rules_closure//closure/protobuf:protobuf_js.BUILD",
+        sha256 = "9111bf0b542b631165fadbd80aa60e7fb25b25311c532139ed2089d76ddf6dd7",
+        strip_prefix = "protobuf-3.18.1/js",
```

Comment:
Last version is 3.19.1 see https://github.com/bazelbuild/rules_closure/pull/539/files

Reply:
Looking at the PR, it seems like it will pull in new deps and that will require additional changes.